before_script_test.local.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
dan, this must be in root folder... we should change this to .travis folder

cc @greg0ire

Reply:
yea i struggled over it :-)